extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/HueConfigStatusMessage.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2014-2016 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.hue.internal;
+
+import org.eclipse.smarthome.config.core.status.ConfigStatusMessage;
+
+/**
+ * The {@link HueConfigStatusMessage} defines
+ * the keys to be used for {@link ConfigStatusMessage}s.
+ * 
+ * @author Alexander Kostadinov - Initial contribution
+ *
+ */
+public enum HueConfigStatusMessage {
+    BRIDGE_NOT_RESPONDING("bridge-unreachable"),
+    IP_ADDRESS_MISSING("missing-ip-address-configuration");
+
+    public static final String CONFIG_STATUS_ERROR_PREFIX = "config.status.error.";
```

Comment:
Yes (+ we would need to mention in the documentation, how the property keys must look like)
Alternatively, we could simply move the constant to the framework and leave it to the implementor, whether he wants to use it or not (and advice using it in the docs).
I leave it up to you what way you prefer!